client/state/reader/reducer.js

```diff
@@ -30,4 +31,5 @@ export default combineReducers( {
 	tags,
 	thumbnails,
 	teams,
+	feedSearch,
```

Comment:
I reckon this should be called `feedSearches`, since all the other reducers are plural.